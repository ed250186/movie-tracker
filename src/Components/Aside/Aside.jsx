import React, {Component} from 'react';
import './Aside.scss';
import active from '../../images/bookmark-active.png';
import inactive from '../../images/bookmark-inactive.png';


class Aside extends Component {
  constructor() {
    super()
    this.state = {
      currentTab: 'home'
    }
  }

  image ={
    height: '50px'
}


  render() {
    return(
      <aside>
        <button>Favorites</button>
        <img style={this.image} src={active} alt=""/>
        <img style={this.image} src={inactive} alt=""/>
      </aside>
    )
  }
}


export default Aside;