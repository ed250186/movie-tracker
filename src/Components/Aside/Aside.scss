aside {
  background: #0F1C40;
  border-right: #485981 solid 1px;
  display: flex;
  justify-content: center;
  min-height: 100%;
  padding-top: 20px;
  width: 15%;
  button {
    cursor: pointer;
    font-size: 20px;
    height: 35px;
    width: 115%;
  }
}