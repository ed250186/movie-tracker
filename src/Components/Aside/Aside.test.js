import React from 'react';
import {shallow} from 'enzyme';
import Aside from './Aside';

describe('App', () => {
  let wrapper, instance;

  beforeEach(() => {
    wrapper = shallow(<Aside/>);
    instance = wrapper.instance();
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

})