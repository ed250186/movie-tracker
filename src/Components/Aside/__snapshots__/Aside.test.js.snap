// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<aside>
  <NavLink
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(FavoriteContainer)",
        "type": [Function],
      }
    }
    to="/favorites"
  >
    <button>
      Favorites
    </button>
  </NavLink>
</aside>
`;
